xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -246,9 +246,10 @@ def set(self, key, structure, course_context=None):
             data_size = len(compressed_pickled_data)
             tagger.measure('compressed_size', data_size)
 
-            # Structures are immutable, so we set a timeout of "never"
+            # We rely on the course structure cache default timeout, which should be
+            # high by default (~ a few days).
             try:
-                self.cache.set(key, compressed_pickled_data, None)
+                self.cache.set(key, compressed_pickled_data)
```

Comment:
@ormsbee @regisb: Have you ensure that this is the only use of `course_structure_cache`, and that the `TIMEOUT` setting wasn't being used in some cases and not others?

Reply:
@robrap: It is. It's a very narrowly-scoped cache in its usage, and this is the only access point.

Reply:
Actually, since I just pushed my setting change to Edge by mistake, I did the search myself, and saw the same. Thanks for confirming @ormsbee.